test/errors/tupleExpandNamedArgs-brief.good

```diff
@@ -0,0 +1,2 @@
+error in tupleExpandNamedArgs.chpl:3: tuple expansion cannot be used with named arguments
```

Comment:
Hmm, that seems wrong/surprising to me.  Specifically, I'd guess that the problem here is that `a` is not a varargs argument; and that passing `a=(...t)` would be OK if `a` were a varargs argument like `proc f(a..., b: int)` (whether or not it works today).  So I think the more accurate error would be "cannot be used to pass values to a non-varargs argument" (though even that is a bit colloquial...  "non-variadic argument" seems more precise, but also less likely to be understood by a naive user).